packages/fscomponents/src/components/Step.tsx

```diff
@@ -53,6 +53,9 @@ export class Step extends Component<StepProps> {
         <TouchableOpacity
           onPress={this.props.onPress}
           style={[styles.container, this.props.style]}
+          accessibilityTraits='button'
```

Comment:
`accessibilityTraits` and `accessibilityComponentType` are deprecated for iOS and Android respectively. Can `accessibilityRole` be used instead?

Reply:
Yes, we can use it but there will be typescript error if that's okay.